mariadb.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "MariaDB",
+  "plugin_schema": "2",
+  "release": "12.2-RELEASE",
+  "artifact": "https://github.com/m1ch/iocage-plugin-mariadb.git",
+  "official": false,
+  "properties": {
+    "allow_raw_sockets": "1",
```

Comment:
Please change the values `1/0` to integer

Reply:
Thanks for the hint. Update done!